### PR TITLE
Task-48942: the comment for the versions is added to the activity of the initial document (copied/pasted)

### PR DESCRIPTION
**Pull request by wassimmhamdi**
Problem: The comment of the version of the documentY is added in the activity of the documentX
Fix: Version comment of documentY are added to the correct activity of documentY

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you rebase your PR and resolv the conflict ?
Thanks

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 1]
```diff
@@ -154,6 +154,7 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
           }else {
             int fileIndex = 0;
             String copyTitle = null;
+            if (isFolder) {
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: Please check formatting (indentation is not ok)

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 2]
```diff
@@ -529,7 +533,14 @@ public static void copyNode(Session session, String srcWorkspaceName, String src
     if (workspace.getName().equals(srcWorkspaceName)) {
       try {
         workspace.copy(srcPath, destPath);
-        Node destNode = (Node) session.getItem(destPath);
+        Node parentDestNode = session.getItem(destPath).getParent();
+        String destName = session.getItem(destPath).getName();
+        NodeIterator destNodeIterator = parentDestNode.getNodes(destName);
+        Node destNode = null;
+
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java:540]: add a test if(destNode!=null) {

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java (hunk 2 above)
Comment (wassimmhamdi) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java:540]: I did it

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 3]
```diff
@@ -529,19 +533,30 @@ public static void copyNode(Session session, String srcWorkspaceName, String src
...
+          }
+          destNode.save();
         }
-        destNode.save();
 
-        if(copyTitle != null)
+        if(copyTitle != null) {
           destNode.setProperty("exo:title", copyTitle);
+        }
         Utils.removeReferences(destNode);
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: this part should be also in the if

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]
